katas/content/multi_qubit_gates/index.md

```diff
@@ -259,7 +269,14 @@ The $SWAP$ gate acts on two qubits, and, as the name implies, swaps their quantu
         </td>
     </tr>
 </table>
-
+@[exercise]({
```

Comment:
After some time glaring at the new and the old tasks, I think we can just keep the old one and not add the new one. It shouldn't need the math of acting on non-adjacent qubits, since they're just swaps - at least I hope so. If we hear back from a confused learner, we'll revisit